dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -760,15 +802,15 @@ public void constantField_notUpdatedCache_whenChanged() throws Exception{
             addAnonymousPermissions(contentlet);
 
             final HTMLPageAsset pageEnglishVersion = new HTMLPageDataGen(folder, template)
-                    .languageId(1).pageURL("testPageWidget").title("testPageWidget")
+                    .languageId(1).pageURL("testPageWidget"+ System.currentTimeMillis()).title("testPageWidget")
                     .nextPersisted();
             pageEnglishVersion.setIndexPolicy(IndexPolicy.WAIT_FOR);
             pageEnglishVersion.setIndexPolicyDependencies(IndexPolicy.WAIT_FOR);
             contentlet.setBoolProperty(Contentlet.IS_TEST_MODE, true);
             contentletAPI.publish(pageEnglishVersion, systemUser, false);
             addAnonymousPermissions(pageEnglishVersion);
 
-            MultiTree multiTree = new MultiTree(pageEnglishVersion.getIdentifier(), containerId,
+            MultiTree multiTree = new MultiTree(pageEnglishVersion.getIdentifier(), container.getIdentifier(),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'multiTree' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5067852)